desktop/context-menu/index.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+/**
+ * External dependencies
+ */
+const { Menu } = require('electron');
+
+const { appCommandSender } = require('../menus/utils');
+
+module.exports = function (mainWindow) {
+  mainWindow.webContents.on('context-menu', (event, params) => {
+    const { editFlags } = params;
+    const template = [
+      {
+        id: 'selectAll',
+        label: 'Select All',
+        click: appCommandSender('editorCommand', { action: 'selectAll' }),
+        enabled: editFlags.canSelectAll,
+      },
+      {
+        id: 'cut',
+        label: 'Cut',
+        role: 'cut',
+        enabled: editFlags.canCut,
+      },
+      {
+        id: 'copy',
+        label: 'Copy',
+        role: 'copy',
+        enabled: editFlags.canCopy,
+      },
+      {
+        id: 'paste',
+        label: 'Paste',
+        role: 'paste',
+        enabled: editFlags.canPaste,
+      },
+      {
+        type: 'separator',
+      },
+    ];
+    const menu = Menu.buildFromTemplate(template);
+    menu.popup({});
```

Comment:
I assume these are split up as part of the development process? if we find it useful we could inline the function. how could it be useful? if we found more value in the brevity than in the verbosity

```js
mainWindow.webContents.on('context-menu', (event, { editFlags }) =>
	Menu.buildFromTempalte([
		{
			…
		},
		…
	]).popup({});
```

I'm assuming that argument-destructuring works since the object destructuring is working.
if you don't think this is a positive change just ignore it.